### ISSUE TITLE
Sound aplet on menu have no a11y description and is unusable with keyboard

### ISSUE THREAD

**Issue by schidlo** [mate-desktop/mate-panel#1491]
On Fedora 42 MATE spin when you start Orca screen reader, go to the top bar and move to sound icon (using tab button) then nothing is read. This mean that Orca's users are lost at what is going on. 
Another problem is that if you use the system just with a keyboard there is nothing you can do with the sound menu. This isn't the primary way how the sound would be managed but it would be great if people could use it if they want.

How to reproduce:

1. Turn Orca on by pressing alt+f2 and writing "orca"
2. Go to top bar with Ctrl+Alt+Tab
3. Use tab to move to sound icon using tab

What happens:
There is just silence. No information provided on what icon is active

What should happen:
There should be information on what kind of menu is active and some basic information - like what is the sound output level. 
Also it would be great if you could do something with that menu just using keyboard.

Version of component:
mate-panel-1.28.4-1.fc42.x86_64

**Comment by kendell-clark** [comment]
I experience this behavior or vojtux 43, a fedora mate spin with accessibility enabled by default and additional software included by default. The original fedora 43 mate spin does not have this issue but vojtux does.
